dbcon/mysql/ha_mcs_sysvars.cpp

```diff
@@ -205,6 +205,7 @@ static MYSQL_THDVAR_STR(cmapi_version, PLUGIN_VAR_NOCMDOPT|PLUGIN_VAR_MEMALLOC,
 
 static MYSQL_THDVAR_STR(cmapi_key, PLUGIN_VAR_NOCMDOPT|PLUGIN_VAR_MEMALLOC, "CMAPI key", NULL, NULL,
                         "");
+static MYSQL_THDVAR_STR(pron, PLUGIN_VAR_MEMALLOC, "Debug options json dictionary", NULL, NULL, "");
```

Comment:
I think `PLUGIN_VAR_NOCMDOPT|PLUGIN_VAR_MEMALLOC` would be a more appropriate flag here.

Reply:
++